Cannot find module 'babel-runtime/helpers/interop-export-wildcard'

Getting this error when running simulator. Reinstalling modules: `rm -rf node_modules && npm install` gives the same error.

I did this:
- `rm -rf node_modules`
- checked if node was still running (was not running)
- restarted my computer
- `npm install`

I'm still getting the same error.  When I run `npm ls --depth=0` I'm getting this:
```
├── babel-core@6.7.4
├── babel-eslint@6.0.2
├── babel-plugin-transform-runtime@6.6.0
├── babel-polyfill@6.7.4
├── babel-preset-react-native@1.5.6
├── babel-runtime@6.6.1
├── chai@3.5.0
├── enzyme@2.2.0
├── eslint@2.7.0
├── eslint-config-airbnb@6.2.0
├── eslint-plugin-import@1.4.0
├── eslint-plugin-react@4.2.3
├── eslint-plugin-react-native@1.0.0
├── estraverse-fb@1.3.1
├── lodash@4.8.2
├── moment@2.12.0
├── react@0.14.8
├── react-addons-test-utils@0.14.8
├── react-dom@0.14.8
├── react-native@0.20.0
├── react-native-mock@0.0.6
└── react-native-simple-store@0.1.0

npm ERR! missing: babel-runtime@^5.0.0, required by babel-code-frame@6.7.4
npm ERR! missing: babel-runtime@^5.0.0, required by babel-generator@6.7.2
npm ERR! missing: babel-runtime@^5.0.0, required by babel-helpers@6.6.0
npm ERR! missing: babel-runtime@^5.0.0, required by babel-messages@6.7.2
npm ERR! missing: babel-runtime@^5.0.0, required by babel-template@6.7.0
npm ERR! missing: babel-runtime@^5.0.0, required by babel-traverse@6.7.4
npm ERR! missing: babel-runtime@^5.0.0, required by babel-types@6.7.2
npm ERR! missing: babel-runtime@^5.0.0, required by babylon@6.7.0
```
(more npm errors removed for brevity)

Trying to run the simulator here returns the babel-runtime error above.  Running `npm install` again, seems like it installs the `babel-runtime` dependencies, and `npm ls --depth=0` doesn't show the npm errors.  The simulator now opens, but still getting the `Uncaught ReferenceError`.

Totally lost 😩

--- a/package.json
+++ b/package.json
@@ -7,9 +7,8 @@
     "test": "mocha --require react-native-mock/mock.js --compilers js:babel-core/register --recursive App/**/__tests__/*.test.js"
   },
   "babel": {
-    "presets": ["react-native"],
-    "plugins": [
-      "transform-runtime"
+    "presets": [
+      "react-native"
     ]
   },
   "dependencies": {
@@ -22,15 +21,13 @@
     "estraverse-fb": "^1.3.1",
     "lodash": "^4.5.0",
     "moment": "^2.12.0",
-    "react-native": "^0.20.0",
+    "react-native": "^0.23.0",
     "react-native-simple-store": "^0.1.0"
   },
   "devDependencies": {
     "babel-core": "^6.7.2",
-    "babel-plugin-transform-runtime": "^6.6.0",
     "babel-polyfill": "^6.7.4",
     "babel-preset-react-native": "^1.5.2",
-    "babel-runtime": "^6.6.1",
     "chai": "^3.5.0",
     "enzyme": "^2.1.0",
     "react": "^0.14.8",

